index.html

```diff
@@ -820,8 +820,13 @@ <h3>
           <li>[=list/For each=] |rawInputIndex:long| of [=the range=] from 0 to
           |inputCount| − 1:
             <ol>
-              <li>If the the gamepad button at index |rawInputIndex|
-              [=represents a Standard Gamepad button=]:
+              <li>Let |button| be the button at index |rawInputIndex|.
```

Comment:
Assigning the button to a variable |button| seems awkward. |button| doesn't have a defined type, it's intended to represent a specific physical button under some device-defined button ordering. Let's reword this to avoid assigning it to anything.

```
<li>Let |type| be the result of determining if the button at index |rawInputIndex|
[=represents a Standard Gamepad button=] or...
</li>
```

Reply:
@nondebug  Thanks Matt for the review. Just updated the PR.